tools/batchTestGenerator/internal/testCaseBuilder.go

```diff
@@ -65,7 +65,7 @@ func buildTestCases(runConfig RunConfig) ([]TestCaseInfo, error) {
 						}
 						newTests = append(newTests, newTest)
 					}
-				case "EKS_FARGATE", "EKS_ADOT_OPERATOR", "EKS_ADOT_OPERATOR_ARM64", "EKS", "EKS_ARM64":
+				case "EKS_FARGATE", "EKS_ADOT_OPERATOR", "EKS_ADOT_OPERATOR_ARM64", "EKS", "EKS_ARM64", "EKS_JAVA_INSTRUMENTATION_ADOT_OPERATOR", "EKS_JAVA_INSTRUMENTATION_ADOT_OPERATOR_ARM64":
```

Comment:
Why do these need to be two new test case types rather than use the existing `EKS_ADOT_OPERATOR` or `EKS_ADOT_OPERATOR_ARM64` types?

Reply:
@bryan-aguilar I noticed that adot-operator workflow is running the `testcases` that are included in the `EKS_ADOT_OPERATOR` or `EKS_ADOT_OPERATOR_ARM64` tags. 
I created the new tag exclusive for java-instrumentation as I wanted to exclude them from all other scopes of adot-Collector and adot-Operator testcases. Reason: To not duplicate the running of same testcases with adot-operator workflow.

Reply:
The test cases that are being ran is dependent on which tests are defined in the `testcases.json` file. If you implement the advice given to you elsewhere, to give the java-instrumeentation it's own `testcases.json` file you can only define the test cases necessary.

Reply:
@bryan-aguilar. I can change that but forgot to mention one primary reason. 
Tag (like `EKS_ADOT_OPERATOR`) is mapped to a cluster. Since java instrumentation tests are deployed on different cluster, so the new `testcases.json` in `java-instrumentation`, will have different cluster values than the other similar files in other repos. I originally thought this is going to be confusing so thought of creating a new tag specific for `java-instrumentation`. 
Do you think its ok to map the `tag` to `cluster` here differently by reusing the `EKS_ADOT_OPERATOR` tag?

Reply:
A `testcases.json` file should represent the test case configuration required for that specific repository. We SHOULD have a specific `testcases.json` file for each repository. We do not now, but will in the near future. 

I think it's more confusing to separate these into different platform types. There is no reason that we could not run java instrumentation tests within the `ADOT_OPERATOR` test case scope right? In the end these are all tests of the operator.